src/Stratis.Features.Diagnostic/Utils/UnityConversionExtension.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace Stratis.Features.Diagnostic.Utils
+{
+    internal static class UnityConversionExtension
```

Comment:
What is this used for?

Reply:
uhm... good point, I'm pretty sure I was using it to format nicely the swagger statistic output but then I chose to display them in raw bytes, so I can remove it